dlls/cbase.h

```diff
@@ -264,7 +264,13 @@ class CBaseEntity
 		return pEnt; 
 	}
 
-	static CBaseEntity *Instance( entvars_t *pev ) { return Instance( ENT( pev ) ); }
+	static CBaseEntity *Instance( entvars_t *pev )
+	{
+		// Fix hornet crash, among other things
+		if (!pev)
+			return Instance(ENT(0));
```

Comment:
How does this change anything though? Is ENT different based on type (int vs. pointer), if so then what does this return?

Reply:
guess i'll just do the pev->attacker !=nullptr blabla thing then

Reply:
done.